src/NHibernate.Test/NHSpecificTest/GH1704/Fixture.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Linq;
+using NUnit.Framework;
+
+namespace NHibernate.Test.NHSpecificTest.GH1704
+{
+	[TestFixture]
+	public class Fixture : BugTestCase
```

Comment:
Due to the test nature, it appears to me no async code re-generation has been run, because for such a test, it should have generated an async counterpart. Please run the async generator. (See [contributing](https://github.com/nhibernate/nhibernate-core/blob/master/CONTRIBUTING.md#regenerate-async-code).)